### PR TITLE
Needed these changes to get it working from scratch

### PR DESCRIPTION
**Pull request by sigurdga**
Partially related to this comment: https://github.com/ksingh7/Try-Ceph/issues/2#issuecomment-134024323

### CODE REVIEW

File: roles/mons/tasks/main.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 
 - name: Start and add that the monitor service to the init sequence
-  service: name=ceph state=started enabled=yes args=mon
```
Comment (sigurdga) [inline roles/mons/tasks/main.yml:34]: Not sure if this gives some side effects, but using "args=mon" makes systemd trying to start up some "mon.service" instead. Removing it and it works.

File: roles/mons/tasks/main.yml [hunk 2]
```diff
@@ -79,7 +79,7 @@
   command: ceph-authtool -n client.radosgw.gateway --cap osd 'allow rwx' --cap mon 'allow rwx' /etc/ceph/ceph.client.radosgw.keyring
 
 - name: Add RGW key to cluster
-  command: ceph -k /etc/ceph/ceph.client.admin.keyring auth add client.radosgw.gateway -i /etc/ceph/ceph.client.radosgw.keyring
```
Comment (sigurdga) [inline roles/mons/tasks/main.yml:82]: Not necessary for the first run, but if you rerun the provision step, this makes it not barf about keys not matching. Importing on each run.

**Comment by ksingh7** [comment]
Hey @sigurdga  thanks a lot for your contribution and changes.

Have you tested these changes ? If yes then i will merge these changes directly without testing them myself :)

**Comment by sigurdga** [comment]
Hi @ksingh7 I tested by having a clean git repo after adding my changes, running the vagrant and opening the interface.

**Pull request merged**